src/util/index.js

```diff
@@ -76,3 +76,11 @@ export const formatFloatToString = (number: number): string => {
 export const hasProp = (obj: any, prop: string): boolean => {
   return {}.hasOwnProperty.call(obj, prop)
 }
+
+export const isDateBiggerThanToday = (newDate: any): boolean => {
+  if (newDate) {
+    return (moment(newDate).isAfter()) ? true : false
```

Comment:
I'm really curious to know why you feel it's necessary to use an Xor statement here?